shop/urls.py

```diff
@@ -0,0 +1,46 @@
+from django.urls import include, path
+
+from shop.views.address import AddressView
+from shop.views.category import CategoryView
+from shop.views.feedback import FeedbackDetail, FeedbackImagesRemover, FeedbackList
+from shop.views.image import ImageView
+from shop.views.order import OrderView
+from shop.views.order_item import OrderItemView
+from shop.views.product import ProductImagesRemover, ProductView
+from shop.views.product_material import ProductMaterialView
+from shop.views.user import UserAddressesView, UserFeedbackView, UserOrdersView, UserView
+
+urlpatterns = [
+    path('addresses/', AddressView.as_view(http_method_names=['get', 'post']), name='address-list'),
+    path('addresses/<int:pk>/', AddressView.as_view(http_method_names=['get', 'put', 'delete']), name='address-detail'),
+    path('auth/', include('rest_framework.urls', namespace='rest_framework')),
+    path('categories/', CategoryView.as_view(http_method_names=['get', 'post']), name='category-list'),
+    path('categories/<int:pk>/', CategoryView.as_view(http_method_names=['get', 'put', 'delete']),
+         name='category-detail'),
+    path('category/<int:category_pk>/', ProductView.as_view(http_method_names=['get']),
+         name='product-list-by-category'),
```

Comment:
what is the difference between these two endpoints?

Reply:
`category/<int:category_pk>/` retrieves a list of products filtered by a specific category